### ISSUE TITLE
Quickview overlay arrows lost at 200% text zoom on desktop

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#16029]
This is a violation of [WCAG S.C. 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)

To reproduce:

- In Firefox go to preferences and search for Zoom
  - tick Zoom text only
- set viewport to `1280x768`
- Zoom to 200%

<img width="2560" height="1536" alt="Image" src="https://github.com/user-attachments/assets/eae4af76-fa7c-44f3-a713-d13e6e779a28" />
